packages/portal-proto/package.json

```diff
@@ -30,7 +30,7 @@
     "@mantine/notifications": "^6.0.21",
     "@react-spring/web": "^9.5.5",
     "@reduxjs/toolkit": "^1.8.5",
-    "@sjcrh/proteinpaint-client": "^2.40.1",
+    "@sjcrh/proteinpaint-client": "^2.40.3",
```

Comment:
This is @sjcrh/proteinpaint-client": "^2.40.2" in package-lock.json and is causing a build error.

Reply:
Fixed.